client/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "watch": "webpack --stdin --progress --color",
     "build": "webpack",
-    "deploy": "webpack -p",
+    "deploy": "webpack -p -d",
```

Comment:
We want to deploy the debug mode with devtools and sourcemap ?

Reply:
No, sorry.

Reply:
Done.